pages/gigabit/Terms.jsx

```diff
@@ -22,120 +22,221 @@ module.exports = React.createClass({
+              </ol>
+              <li>Organizational Fit (250 Words or Less)</li>
+              <ol>
+                <li>Describe how this project fits into your organization’s mission and goals.</li>
+              </ol>
+              <li>Internet Health</li>
+              <ol>
+                <li>With which Internet health issue(s) does your project align? How does it help to build a healthier future internet?</li>
+              </ol>
+              <li>Project History (250 Words or Less)</li>
+              <ol>
+                <li>Is this a new project or a continuation? If it’s new, please describe your qualifications to initiate the activity. If continued, please describe your accomplishments to date. Feel free to include links to articles and documents online which highlight your recent work.</li>
+              </ol>
+              <li>Sustainability (250 Words or Less)</li>
+              <ol>
+                <li>How will you continue work on this project beyond this funding period?</li>
+                <li>How will you encourage other organizations to adopt this program?</li>
+                <li>How might this work be scaled and spread to other organizations or to other cities?</li>
+              </ol>
+              <li>Previous Work with Mozilla: (250 Words or Less)</li>
+              <ol>
+                <li>Have you previously partnered with Mozilla? Have you previously received a grant from Mozilla? If so, please list title, amount, purpose.</li>
+              </ol>
             </ol>
+
             <p>
-            Each Submission must comply with any word limits or other terms or requirements on the Fund website. The Sponsor, at its sole discretion, reserves the right to not consider any Submission not in compliance with these Terms or for any reason.
+              A <a href="https://docs.google.com/spreadsheets/d/1WxZuMHz7JlcmkcVP4i-DCx7Ulr5NvNq1pSK2MgqOVbQ/edit">project budget</a>, <a href="https://drive.google.com/open?id=1Yiam58szxLTYVVSluN8zP93LgB2Z3Ulah2ZzqknRB6c">project timeline</a>, and letters of support from key collaborators must also be submitted via mozilla.fluxx.io. Templates for the 16-week project timeline and budget are available on the Mozilla Gigabit website.
             </p>
             <p>
-            Submissions and any subsequent communications or postings associated with the Fund may not contain advertisements, commercial endorsements, or any material that is indecent, obscene, hateful, discriminatory, defamatory, slanderous, libelous, unlawful or otherwise inappropriate as determined by Sponsor in its absolute discretion. Any inappropriate Submissions or material may be disqualified by Sponsor in its absolute discretion.
+              The following organizational information must be submitted along with your project description via mozilla.fluxx.io:
+            </p>
+            <ol>
+              <li>Required of all Non-Profit Applicants</li>
```

Comment:
I think the `Required of ...` lines should be `<p`> and not bullet points.

```jsx
<p><u>Required of all Non-Profit Applicants</u></p>
<ol>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ol>

<p><u>Required of all For-Profit Applicants</u></p>
<ol>
  <li>...</li>
  <li>...</li>
  <li>...</li>
</ol>
```

It reads better this way. What do you think? @hannahkane